### ISSUE TITLE
Got here from optipng-bin, which says I want to use this... but why? 

### ISSUE THREAD

**Issue by Pomax** [imagemin/imagemin-optipng#25]
https://github.com/imagemin/optipng-bin says "You probably want `imagemin-optipng` instead" but I don't see anything that shows this can be used as CLI utility.... why should we want to use this?

**Comment by ogonkov** [comment]
I guess `optipng-bin` is low-level package that is not recommended for direct use. This package looks like user-friendly wrapper for `optipng-bin`.

**Comment by Pomax** [comment]
Guesses don't help people understand why they're being told to switch package. Low level is fine for automated cleanup, so the original question remains: why is the readme.md saying what it says?

**Comment by 1000ch** [comment]
`optipng-bin` is a low-level module, and `imagemin-optipng` provides a nicer interface by wrapping `optipng-bin`. Which we want to use depends on our use-case, so it says "You probably want `imagemin-optipng` instead."

**Comment by Pomax** [comment]
Right, but this is an `npm` package, hosted on `github`, and uses signaling akin to what people use to indicate that a package is deprecated. Rather than saying "you probably want to use X" (which in the context of an npm package is just a polite way to say "stop using this") it would be better to add a heading an one or two paragraphs that explain when you want to use `optipng-bin`, and when it makes more sense to use `imagemin-optipng`.

We use it for CLI cleaning of PNG files, for instance, where we _definitely_ don't want a more elaborate interface.
